wagtailio/blog/models.py

```diff
@@ -124,6 +157,22 @@ def siblings(self):
         + [FieldPanel("canonical_url")]
     )
 
+    @cached_property
+    def related_pages(self):
+        return self.page_related_pages.all()
+
+    @cached_property
+    def meta_text(self):
+        return self.category.title
+
+    @cached_property
+    def meta_icon(self):
+        return self.category.icon
```

Comment:
question: `self.category` can be `None`. what happens when you access a blog post with no category?

Reply:
Good shout! Fixed in f225dad. Also updated `meta_text` accordingly